admin/base/settings/defaults.py

```diff
@@ -23,9 +25,31 @@
     '.osf.io'
 ]
 
+# AUTH_PASSWORD_VALIDATORS available only in Django 1.9
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+        'OPTIONS': {
+            'min_length': 5,
+        }
+    },
+]
+
 # Email settings. Account created for testing. Password shouldn't be hardcoded
+# [DEVOPS] this should be set to 'django.core.mail.backends.smtp.EmailBackend' in the > dev local.py.
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
+# Sendgrid Email Settings - Using OSF credentials
+
+EMAIL_HOST = 'smtp.sendgrid.net'
+EMAIL_HOST_USER = 'osf-smtp'
+EMAIL_HOST_PASSWORD = osf_settings.MAIL_PASSWORD  # local.py
```

Comment:
If we're going to delegate the password to `website.settings`, let's do the same with host and username.